proc_Hahn_echoph.py

```diff
@@ -1,67 +1,45 @@
 from pyspecdata import *
 from scipy.optimize import leastsq,minimize,basinhopping
-fl = figlist_var()
+from proc_scripts import *
+from proc_scripts.load_data import postproc_lookup
+from sympy import symbols
+fl = fl_mod()
 t2 = symbols('t2')
 filter_bandwidth = 5e3
-for date,id_string,label_str in [
-        ('191007','echo_2','n'),
+color_choice = True
+for searchstr,exp_type,nodename,postproc,label_str,color_str in [
+        ('200302_alex_probe_water','test_equip','signal','Hahn_echoph','microwaves off','blue'),
```

Comment:
cool.  If I'm guessing right as to what's happening below, you could also just run the `color_choice == False` code when the color is set to `None`  that eliminates the need for line 9

Reply:
1. "what's happening below"= the processing? or the loading of data?
2. where is the color set to none in this code? I don't see it...

Reply:
for example, you can replace `'blue'` with `None` then, later in the code, rather than checking if `color_choice` is `False` you can check if `color_str` is `None`

Reply:
done

Reply:
actually, looking at the code, neither color_str nor color_choice are used?

Reply:
correct. only in the beginning when loading the file. So I am just going to get rid of those....

Reply:
👍